layers/meta-resin-beaglebone/conf/machine/beaglebone-pocket.conf

```diff
@@ -0,0 +1,9 @@
+#@TYPE: Machine
+#@NAME: PocketBeagle
+#@DESCRIPTION: Machine configuration for the PocketBeagle
+
+MACHINEOVERRIDES = "beaglebone:${MACHINE}"
+
+include conf/machine/beaglebone.conf
+
+KERNEL_DEVICETREE_beaglebone-pocket = "am335x-pocketbeagle.dtb "
```

Comment:
It's safe to use here just `KERNEL_DEVICETREE` since you are in the machine configuration file

Reply:
Left it as is because without it the dtb didn't get generated for this machine.